.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Let's run this on the following versions:
```suggestion
    runs-on: ubuntu-latest
    strategy:
      matrix:
        python-version:
          - "3.6"
          - "3.7"
          - "3.8"
          - "3.9"
          - "3.10"
          - "3.11"
```

---

@antoniagaete it looks like some of those versions can only be run on some ubuntu platforms unfortunately. You might need to split this job up or remove some of those versions to make this simpler.

---

I ended up removing them, but let me know if you want me to split up the job instead

---

Let's just test for 3.6, 3.9 and 3.10